neqo-transport/src/send_stream.rs

```diff
@@ -438,7 +438,7 @@ pub struct SendStream {
 }
 
 impl SendStream {
-    pub fn new(
+    pub(crate) fn new(
```

Comment:
If the type is not published (either by being `pub(crate)` itself, or by the module not being `pub use send_stream;`), you don't need to make functions this way.

Maybe we should have a style guide option here.

Reply:
LOL, this isn't me just being arbitrary, back at commit 1c6544a (i.e. before merge of Ignition's changes) *not* putting pub(crate) caused:
```
error[E0446]: crate-visible type `events::ConnectionEvents` in public interface
   --> neqo-transport/src/send_stream.rs:441:5
    |
441 | /     pub fn new(
442 | |         stream_id: StreamId,
443 | |         max_stream_data: u64,
444 | |         flow_mgr: Rc<RefCell<FlowMgr>>,
...   |
456 | |         }
457 | |     }
    | |_____^ can't leak crate-visible type
    | 
   ::: neqo-transport/src/events.rs:48:1
    |
48  |   pub(crate) struct ConnectionEvents {
    |   ---------- `events::ConnectionEvents` declared as crate-visible
```
Since they're now no longer needed I will delete them.